File: Linguagem Lua/cria.lua
-- Criacao de arquivo texto 

ARQ = io.open("teste.plt","w")
ARQ:write("reset\n")
ARQ:write("set grid\n")
ARQ:write("plot gauss(x,-2,2) title \"Gauss em Lua\"\n")
ARQ:write("pause -1\n")
ARQ:close()
os.execute("gnuplot teste.plt")

File: Linguagem C/cria.c
// Criação de arquivo texto 

#include <stdio.h>
#include <stdlib.h>

int main(void)
{
  FILE *ARQ;
  ARQ = fopen("teste.plt", "w");
  fprintf(ARQ,"reset\n");
  fprintf(ARQ,"set grid\n");
  fprintf(ARQ,"plot gauss(x,-2,2) title \"Gauss em C\"\n");
  fprintf(ARQ,"pause-1\n");
  fclose(ARQ);
  system("gnuplot teste.plt");
  return 0;
}
